docs/guides/upgrading.rst

```diff
@@ -52,6 +52,15 @@ New API for handling entity icons
  * ``ElggEntity::hasIcon()`` - checks if an icon with given size has been created
  * ``elgg_get_embed_url()`` - can be used to return an embed URL for an entity's icon (served via `/serve-icon` handler)
 
+New API for page and action handling
+------------------------------------
+
+Page handlers and action files can now return an instance of ``\Elgg\Http\ResponseBuilder``.
+Plugins can use the following convenience functions:
+
+ * ``elgg_ok()`` sends a 2xx response with HTML (page handler) or JSON data (actions)
```

Comment:
What happens if an action returns this? We always redirect by default, do we override that?